src/control-plane/backend/lambda/api/service/quicksight/sql-builder.ts

```diff
@@ -149,12 +148,12 @@ export const BUILTIN_EVENTS = [
 ];
 
 export enum ExploreAnalyticsType {
-  FUNNEL = 'is_null',
-  EVENT = 'is_not_null',
-  EVENT_PATH = '=',
-  NODE_PATH = '<>',
-  RETENTION = '>',
-  ATTRIBUTION = '>=',
+  FUNNEL = 'FUNNEL',
```

Comment:
do we have tests covering this change?

Reply:
just used in sql-builder.ts.  already checked.